misc/dbt-materialize/mzcompose.yml

```diff
@@ -19,18 +20,49 @@ services:
       - MZ_DEV=1
     ports:
       - *materialized
+    depends_on: [test-certs]
+  tls-materialized:
+    mzbuild: materialized
+    command:
+      - --disable-telemetry
+      - --tls-mode=verify-ca
+      - --tls-cert=/share/secrets/materialized.crt
+      - --tls-key=/share/secrets/materialized.key
+      - --tls-ca=/share/secrets/ca.crt
+      - --listen-addr=0.0.0.0:6876
```

Comment:
nit: I'm not sure how `x-port-mapping` works, but hard-coding the port here might break it. If so, I'd be OK with removing the `x-port-mapping` and just hard code the ports.

```
diff --git a/misc/dbt-materialize/mzcompose.yml b/misc/dbt-materialize/mzcompose.yml
index d7d29ed53..aa2643d8c 100644
--- a/misc/dbt-materialize/mzcompose.yml
+++ b/misc/dbt-materialize/mzcompose.yml
@@ -7,10 +7,6 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.

-x-port-mappings:
-  - &materialized ${MZ_PORT:-6875:6875}
-  - &tls-materialized ${TLS_MZ_PORT:-6876:6876}
-
 version: '3.7'
 services:
   materialized:
@@ -19,8 +15,7 @@ services:
     environment:
       - MZ_DEV=1
     ports:
-      - *materialized
-    depends_on: [test-certs]
+      - 6875
   tls-materialized:
     mzbuild: materialized
     command:
@@ -35,7 +30,7 @@ services:
     environment:
       - MZ_DEV=1
     ports:
-      - *tls-materialized
+      - 6876
     depends_on: [test-certs]
   dbt-test:
     mzbuild: dbt-materialize
@@ -49,9 +44,6 @@ services:

 mzworkflows:
   ci:
-    env:
-      MZ_PORT: 6875
-      TLS_MZ_PORT: 6876
     steps:
       - step: start-services
         services: [materialized, tls-materialized]
--
2.25.1
```

Reply:
Happy to remove the `x-port-mapping`! For my understanding, what is it that could break here?

Reply:
It's a constructed case, but an update to `TLS_MZ_PORT` would also require an update to the `--listen-addr` argument. That's why I thought we might as well remove it.